superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -309,6 +403,7 @@ const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
       case MENU_KEYS.EXPLORE_CHART:
         // eslint-disable-next-line no-unused-expressions
         props.logExploreChart?.(props.slice.slice_id);
+        window.open(props.exploreUrl);
```

Comment:
Why do we remove a link and use `window.open` instead? The previous approach had all the benefits of using a normal link, like right click context menu

Reply:
When tabbing, the user will land on the Menu item (the li) and then would need to tab again to hit the link in order to open it. This way the action for enter is on the Menu item, so only one tab is needed. It makes sense to have the other context menu benefits, etc, but it's actually really difficult to right click on that element because it gets picked up by the context menu in the chart below. If you have any suggestions about how to add it back though, lmk.

Reply:
In that case, maybe we could set tabIndex=-1 on that particular menu item, so that tab would focus on the link?